File: testprojects/python_main/pipelines/DONOT_OPEN_PYTHON/code/pythonorganization/donot/openme12/graph/SubGraph_2/Script_5.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from .config import *
from pythonorganization.donot.openme12.udfs.UDFs import *

def Script_5(spark: SparkSession, in0: DataFrame) -> DataFrame:
    from scipy.special import cbrt
    cb = cbrt([27, 64])
    print(cb)
    import torch
    import math
    dtype = torch.float
    device = torch.device("cpu")
    x = torch.linspace(- math.pi, math.pi, 2000, device = device, dtype = dtype)
    y = torch.sin(x)
    a = torch.randn((), device = device, dtype = dtype)
    print(a)
    out0 = in0

    return out0

File: testprojects/scala_main/pipelines/SCALA_DEP_MGMT_ALL/code/src/main/scala/org/main/scla_dep_mgmt/graph/Script_5.scala
package org.main.scla_dep_mgmt.graph

import io.prophecy.libs._
import org.main.scla_dep_mgmt.config.ConfigStore._
import org.main.scla_dep_mgmt.udfs.UDFs._
import org.main.scla_dep_mgmt.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Script_5 {
  def apply(spark: SparkSession, in0: DataFrame): DataFrame = {
    // import epic.models.{ParserSelector}
    // val parser = epic.models.ParserSelector.loadParser("en")
    // print(parser)
    
    import cats.syntax.show._
    val shownInt = 123.show 
    print(shownInt)
    
    var out0=in0.select("c_array--long")
    out0
  }

}
